tools/process-svgs.js

```diff
@@ -1,81 +1,83 @@
+                multipass: true,
+                plugins: ["preset-default", "inlineStyles"],
+            });
 
-      // Replace <svg> and </svg> tags
-      svgs[folder][key] = result.data
-        .replace(/.*<svg.*?>/, "")
-        .replace("</svg>", "");
+            // Replace <svg> and </svg> tags
+            svgs[folder][key] = result.data
+                .replace(/.*<svg.*?>/, "")
+                .replace("</svg>", "");
+        }
     }
-  }
 
-  fs.writeFileSync(
-    path.join(__dirname, "..", "src", "svgs.ts"),
-    `${warning}\n\nexport default ${JSON.stringify(svgs)};`,
-  );
+    fs.writeFileSync(
+        path.join(__dirname, "..", "src", "svgs.ts"),
+        `${warning}\n\nexport default ${JSON.stringify(svgs)};`,
+    );
 
-  const svgsIndex = {
-    ...svgs,
-  };
-  for (const key of Object.keys(svgsIndex)) {
-    svgsIndex[key] = Object.keys(svgsIndex[key]);
-  }
-  const svgsGenders = {
-    ...svgsIndex,
-  };
-  for (const key of Object.keys(svgsGenders)) {
-    const keyGenders = [];
-    for (const featureName of svgsGenders[key]) {
-      let gender = genders[key][featureName];
-      if (gender === undefined) {
-        console.log(`Unknown gender for ${key}/${featureName}`);
-        gender = "female";
-      }
-      keyGenders.push(gender);
+    const svgsIndex = {
+        ...svgs,
+    };
+    for (const key of Object.keys(svgsIndex)) {
+        svgsIndex[key] = Object.keys(svgsIndex[key]);
     }
-    svgsGenders[key] = keyGenders;
-  }
-  fs.writeFileSync(
-    path.join(__dirname, "..", "src", "svgs-index.ts"),
-    `${warning}\n\nexport const svgsIndex = ${JSON.stringify(
-      svgsIndex,
-    )};\n\nexport const svgsGenders = ${JSON.stringify(svgsGenders)};`,
-  );
+    const svgsGenders = {
+        ...svgsIndex,
+    };
+    for (const key of Object.keys(svgsGenders)) {
+        const keyGenders = [];
+        for (const featureName of svgsGenders[key]) {
+            let gender = genders[key][featureName];
+            if (gender === undefined) {
+                console.log(`Unknown gender for ${key}/${featureName}`);
+                gender = "female";
+            }
+            keyGenders.push(gender);
+        }
+        svgsGenders[key] = keyGenders;
+    }
+    fs.writeFileSync(
+        path.join(__dirname, "..", "src", "svgs-index.ts"),
+        `${warning}\n\nexport const svgsIndex = ${JSON.stringify(
+            svgsIndex,
+        )};\n\nexport const svgsGenders = ${JSON.stringify(svgsGenders)};`,
+    );
 
-  console.log(
-    `Wrote new src/svgs.ts and src/svgs-index.ts at ${new Date().toLocaleTimeString()}`,
-  );
+    console.log(
+        `Wrote new src/svgs.ts and src/svgs-index.ts at ${new Date().toLocaleTimeString()}`,
+    );
 };
 
-if (require.main === module) {
-  processSVGs();
-} else {
-  module.exports = processSVGs;
-}
+// if (require.main === module) {
+//     processSVGs();
+// } else {
+//     module.exports = processSVGs;
+// }
```

Comment:
Deleting this means that running `node tools/process-svgs.js` will no longer do anything. idk if there is a way to do this same logic with ESM, but if not, we probably need to separate this into two files (one with just the module export, and one that runs on the CLI).